evap/staff/tests/test_importers.py

```diff
@@ -368,12 +369,27 @@ def test_emails_are_cleaned(self):
         self.assertTrue(UserProfile.objects.filter(email="cleaned_diam.synephebos@institution.example.com").exists())
         self.assertTrue(UserProfile.objects.filter(email="cleaned_111@institution.example.com").exists())
 
+    def test_import_with_empty_excel(self):
+        importer_log = import_enrollments(self.empty_excel_content, self.semester, None, None, test_run=True)
+
+        success_messages = [msg.message for msg in importer_log.success_messages()]
+        self.assertIn("The import run will create 0 courses/evaluations and 0 users.", success_messages)
```

Comment:
Please also assert that there were no errors for both assertions in this test (`self.assertEqual(importer_log.errors_by_category(), {})`). In #1855 I came across some tests that still pass, even if the importer has a critical failure, simply because the message was emitted before the fatal failure. I'd like the tests to catch that.